.github/workflows/ci.yaml

```diff
@@ -9,7 +9,10 @@ on:
 
 jobs:
   pathogen-ci:
-    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@dec0880059017dac7facf100435c5737bf1386c8
+    with:
+      workflow-root: phylogenetic
```

Comment:
This is still using the version which was reverted.

Reply:
Thanks, yes I'm aware, worst case is CI fails but as this is under active development right now that's ok. Alternative is to disable pathogen-repo-ci and roll it on its own, but that's not worth the effort